Assets/Tests/InputSystem/Plugins/UITests.cs

```diff
@@ -1793,13 +1793,15 @@ public IEnumerator UI_CanReleaseAndPressTouchesOnSameFrame()
                 .Matches((UICallbackReceiver.Event e) => e.pointerData.pointerType == UIPointerType.Touch).And
                 .Matches((UICallbackReceiver.Event e) => e.pointerData.position == secondPosition));
 
+#if UNITY_2021_2_OR_NEWER
```

Comment:
`EventType.PointerMove` was wrapped in `#if UNITY_2021_2_OR_NEWER` in its declaration.